extensions/grub-arm.sh

```diff
@@ -0,0 +1,101 @@
+# This runs *after* user_config. Don't change anything not coming from other variables or meant to be configured by the user.
+	function extension_prepare_config__prepare_grub-arm() {
```

Comment:
I didn't yet diff with the original.
Fact is original also works on arm64.
So **for clarity** please rename this to `grub-sbc-media` or `grub-rockchip64` or some more clear way of indicating.

Reply:
> Fact is original also works on arm64.

You're wrong, it doesn't work :) The whole nuance is in how you test. You are running tests on a device where you HAVE ALREADY changed the u-boot. Try to do the right test and find that nothing works. For example, take the Rk3399 device from u-boot to eMMC (I pay attention, the device must be in its ORIGINAL state, as it leaves the factory). There is no EFI support in the factory u-boot-2017. :) In addition, there is no u-boot in your original extension, there are no DTBS for the correct operation of the kernel (I have already explained to you that this key difference is the correct use of DTB from the kernel itself).

Reply:
When I say it "works", I mean on actual UEFI firmware, not uboot on SBCs. Neoverse. Ampere. Pythium. SolidRun. many others. Pay attention

Reply:
Anyway since now we have a _clear_ indication of the purpose of the extension let's go!

Reply:
> When I say it "works", I mean on actual UEFI firmware, not uboot on SBCs. Neoverse. Ampere. Pythium. SolidRun. many others. Pay attention

It is commendable that you know well the equipment of different levels for ARM. Do you have this equipment in stock ? Or are you ready to give examples of real use of Armbian-EFI on at least 10 real instances of these models ? :) (the question is rhetorical, no answer is required)